src/utils/setup/create-article-node.js

```diff
@@ -10,13 +10,17 @@ const createArticleNode = (
 ) => {
     const filename = getNestedValue(['filename'], doc) || '';
     const isArticlePage =
-        !filename.includes('images/') && filename.endsWith('.txt');
+        !filename.includes('images/') &&
+        filename.endsWith('.txt') &&
+        // Remove extraneous src/index.txt file in devhub-content repo
+        filename !== 'index.txt';
```

Comment:
There is another piece of odd data in the content repo, `src/index.txt`. I asked Sophie and this isn't used as an article so we should exclude it. Its contents are empty.

Reply:
This is good, but they should probably just remove this file rather than having us exclude extraneous files. I'll ask about this one.